README.md

```
@@ -107,13 +107,13 @@ jobs:
 
 ### `merge-base`
 
-- **Description**: check current branch is rebased onto target branch.
+- **Description**: check current branch is rebased onto the target branch.
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: asterisk; Actual: dash

(MD004, ul-style)